components/molecules/imageLocation/__tests__/stories.js

```diff
@@ -24,67 +26,71 @@ import '../__resources__/images/second-floor.png'
 import '../__resources__/images/third-floor.png'
```

Comment:
@esuarezz We dont want these images in the bundle anymore - load from .storybook/static folder now